routes/home.js

```diff
@@ -24,9 +24,20 @@ module.exports = async (req, res) => {
       );
     }
 
+    const categories = Object.keys(
+      events
+        .reduce((acc, { categories }) => [...acc, ...categories], [])
+        .reduce((acc, category) => {
```

Comment:
You can reduce this to a single reduce using `set`, but I have a feeling babel will make it very long so it's up to you:

```js
events.reduce((acc, { categories }) => [...new Set([...acc, ...categories])], [])
```

Reply:
So you decided on the old method..?

Reply:
yeah